data/metadata.json

```diff
@@ -1,32 +1,38 @@
 {
```

Comment:
Can we sort the file so the diff is a bit smaller?

Reply:
The script sorts it so theoretically after the first merge, it will be a lot nicer: `f.write(json.dumps(metadata, indent=4, sort_keys=True))`